src/s2geography/projections.cc

```diff
@@ -0,0 +1,16 @@
+
+#include "s2/s2projections.h"
+
+namespace s2geography {
+
+S2::Projection* lnglat() {
+  static S2::PlateCarreeProjection projection(180);
+  return &projection;
+}
+
+S2::Projection* mercator() {
+  static S2::MercatorProjection projection(20037508.3427892);
```

Comment:
Is there a need to make this configurable? (maybe since it is in R `s2`?) Currently just took the value from the `s2` code, which I think matches the reported projected bounds of EPSG:3857 (https://epsg.io/3857)